easybuild/easyconfigs/f/flook/flook-0.8.1_flook.pc.in.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/flook.pc.in b/flook.pc.in
```

Comment:
@ahkole Patch description is missing, do you mind opening a follow-up PR that add a short description of why this patch is needed on top, and mentions yourself as author (or includes a reference to source of the patch)?

Note that this implies updating the checksum of the patch in the `flook` easyconfig as well.

Reply:
@boegel see https://github.com/easybuilders/easybuild-easyconfigs/pull/19293